cmd/engine/logger.go

```diff
@@ -27,7 +27,7 @@ func init() {
 	}
 
 	tel = telemetry.New(false)
-	tel.SetRunID("00000000-0000-0000-0000-000000000000")
+	tel.SetRunID("")
```

Comment:
it's not necessary to set it to `""` here since it'll be an empty string anyways, correct?